NetKAN/ROLib.netkan

```diff
@@ -0,0 +1,32 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "ROLib",
+    "$kref"        : "#/ckan/github/KSP-RO/ROLibrary",
+    "$vref"        : "#/ckan/ksp-avc",
+    "abstract"     : "ROLib contains the common code supporting multiple RO* modules",
+    "license"      : "CC-BY-SA-4.0",
+    "author"       : ["pap1723","KSP-RO"],
+    "resources" : {
+        "bugtracker"   : "https://github.com/KSP-RO/ROLibrary/issues",
+        "repository"   : "https://github.com/KSP-RO/ROLibrary"
+    },
+    "tags" : [
+        "library",
+        "plugin"
+    ],
+    "depends" : [
+        { "name" : "ModuleManager" },
+        { "name" : "TexturesUnlimited" }
+    ],
+    "recommends" : [
+        { "name" : "ROEngines" },
+        { "name" : "ROTanks" },
+        { "name" : "ROSolar" },
+        { "name" : "ROCapsules" },
+        { "name" : "RealismOverhaul" },
+        { "name" : "RP-0" }
+    ],
+    "conflicts" : [
+        { "name" : "TweakableEverything" }
```

Comment:
```suggestion
        { "name" : "TweakableEverythingCont" }
```

`TweakableEverything` isn't listed, but `TweakableEverythingCont` is.

Reply:
Thanks for catching that.  Will make sure that gets incorporated into later PRs. :)